source/educators/references/course_development/course_outline.rst

```diff
@@ -45,4 +45,24 @@ following message is visible.
 
 To add content, you :ref:`create a section<Create a Section>`.
 
-
+.. seealso::
+ :class: dropdown
+ 
+ :ref:`Getting Started with Course Content Development` (reference)
```

Comment:
```suggestion
 :ref:`Getting Started with Course Content Development` (reference)
 
  :ref:`Course Outline` (concept)
  
 :ref:`Creating a New Course in Studio <Creating a New Course>` (how-to)
 
 :ref:`Create a Course` (how-to)
 
 :ref:`Create the Course About Page` (how-to)
 
 :ref:`Understanding a Course Outline <Understanding Your Course Outline>` (reference)
 
 :ref:`Add Content in the Course Outline` (reference)
 
 :ref:`Modify Settings for Objects in the Course Outline` (reference)
 
 :ref:`Publish Content from the Course Outline` (reference)
 
 :ref:`Developing Course Sections` (reference)
 
 :ref:`Developing Course Subsections` (reference)
 
 :ref:`Create a Section` (how-to)
 
 :ref:`Create a Subsection` (how-to)
 
 :ref:`Hiding a Subsection from Learners <Hide a Subsection from Students>` (how-to)
 
 :ref:`Add Course Metadata` (how-to)
 
 :ref:`Use a Section from a Course independently of the Course Outline` (how-to)
 
 :ref:`Resources for edx org Course Teams` (reference)
 
 :ref:`Resources for Open edX` (reference)
 
```

Reply:
Resolved spacing issues in See Also section.